.vscode/launch.json

```diff
@@ -28,6 +28,27 @@
           "to": "external/gazelle_cc+"
         },
       ]
+    },
+    {
+      "name": "BCR indexer",
+      "type": "go",
+      "request": "launch",
+      "mode": "exec",
+      "program": "${workspaceFolder}/bazel-bin/index/internal/bcr/indexer/indexer_/indexer",
+      "args": [
+        "--output-mappings=language/cc/bzldep-index.json",
```

Comment:
If it's for debugging purpouse then we might just use default output - it would be stored in current directory (under bazel sandboxing)